### DISCUSSION TITLE
Tonträger - Fragen zum METS-Datenprofil

### DISCUSSION THREAD

**Discussion by maria-federbusch** [kitodo/kitodo-production#5918] [Q&A]
1. Laut Strukturdatenset gibt es bisher keine Dokumenttypen bzw. spezielle Benennungen für Tonträger. Sollte es welche geben? Track ist laut Profil in Analogie zur Seite definiert, weitere spezielle Informationen konnte ich nicht finden. Vgl. https://dfg-viewer.de/fileadmin/groups/dfgviewer/METS-Anwendungsprofil_2.3.1.pdf S. 13
2. Wenn wir sowohl Bilder (Cover, Begleithefte) als auch die Tonträger selbst digitalisieren und in einem digitalen Objekt halten wollen, wie wäre dann die physicalSequence konkret auszugestalten? Gibt es bereits Beispiele in der Community?
3. Wenn der DFG-Viewer audiofähig ist, was erwartet er dann bzgl. der structure Map type Physical?

**Comment by markusweigelt** [comment]
> 1. Laut Strukturdatenset gibt es bisher keine Dokumenttypen bzw. spezielle Benennungen für Tonträger. Sollte es welche geben? Track ist laut Profil in Analogie zur Seite definiert, weitere spezielle Informationen konnte ich nicht finden. Vgl. https://dfg-viewer.de/fileadmin/groups/dfgviewer/METS-Anwendungsprofil_2.3.1.pdf S. 13

Laut meinen Informationen ist das Strukturdatenset (https://dfg-viewer.de/strukturdatenset) keine standardisierte Liste der anzuwendenden Strukturelemente, sondern eine Liste der übersetzten Strukturelement-Bezeichnungen im DFG-Viewer. Der DFG-Viewer kann auch andere Strukturelemente anzeigen.

Wir sind gerade intern in der Abstimmung für welche Tonträger welchen Dokumenttypen bekommen sollen. Dazu werden in unserem Fall die Felder 0500 (Pica3) und 1131 (Pica3) und/oder 1140 (Pica3) aus dem K10Plus/SWB ausgewertet und auf den entsprechenden Dokumenttypen gemappt:
-  audio
-  video
-  multivolume_work
-  volume

> 2. Wenn wir sowohl Bilder (Cover, Begleithefte) als auch die Tonträger selbst digitalisieren und in einem digitalen Objekt halten wollen, wie wäre dann die `physicalSequence` konkret auszugestalten? Gibt es bereits Beispiele in der Community?

An der SLUB berücksichtigen wir derzeit noch keine Begleithefte und Cover, sondern nur rein die Digitalisierung der Tonträger. Sicher wäre das hier perspektivisch sinnvoll.
Aus der Sicht von Kitodo.Produktion sehe ich hier erstmal keine Probleme, Tonträger und deren Begleitmaterialen gemeinsam in einem Vorgang zu erfassen. Bilder z.B. des Begleitmaterials werden nach dem Hochladen oder Ablegen im Vorgangsordner im Metadateneditor als `page` erkannt und Audio oder Video als `track`. 
_Die Erkennung des Typs erfolgt über den Namen. Das heißt Kitodo.Production schaut nach, welche Media Varianten es zu einem Namen in den verschiedenen Unterordnern eines Vorgangs gibt. Falls innerhalb dieser sogenannter Media Varianten keine Datei mit der Media Type Startbezeichnung  "audio" oder "video" gibt, wird die Physische Division mit dem Typen `page` gekennzeichnet ansonsten mit dem Typen `track`._

Mehr Informationen zu den aktuellen Möglichkeiten hinsichtlich AV im Kitodo.Produktion finden Sie in dieser Doku.
https://github.com/kitodo/kitodo-production/wiki/Verwendung-von-Audio-oder-Video

> 3. Wenn der DFG-Viewer audiofähig ist, was erwartet er dann bzgl. der structure Map type Physical?

Ich denke, hierfür wäre sinnvoll eine Diskussion im DFG-Viewer Repository anzustoßen, ein Issue zu öffnen https://github.com/slub/dfg-viewer oder ggf. Sebastian Meyer direkt zu kontaktieren.

**Comment by andre-hohmann** [comment]
Vielen Dank für die Diskussion!

Bezüglich des Strukturdatensets und der möglichen Dokument-/Medientypen möchte ich gerne weitere Interessierte dazu einladen, sich an deren Bestimmung/Bezeichnung zu beteiligen. 
Wenn sich im Kitodo-Anwenderbereich auf Werte für die Dokument-/Medientypen geeinigt werden könnte, wäre dies aus meiner Sicht eine gute Grundlage, um dies gegebenenfalls mit der DDB oder anderen Akteuren abzustimmen. 

Auch wenn wir in der SLUB Dresden zunächst in der Testphase die von @markusweigelt genannten "eigene" Strukturtypen bestimmen und anwenden, wollen wir in Zukunft selbstverständlich auch standardisierte Werte anwenden - sofern diese verfügbar sind. 

Unter Umständen können in diesem Zuge auch Strukturen und Abläufe entwickelt werden, um Strukturdaten/Dokumenttypen-Bezeichnungen abstimmen zu können - ich habe auch den Eindruck, dass diese bisher nicht ausreichend abgebildet sind. 
Diese Diskussion ist aus meiner Sicht ein guter erster Schritt, um auf die Bedarfe hinzuweisen und hoffe auf rege Beteiligung.

**Reply by maria-federbusch** [reply]
Ich habe erste METS-Dateien gebastelt und verwende jetzt als doch recht allgemeinen Vorschlag zur Diskussion:
sound_carrier

**Comment by stefanCCS** [comment]
Vielleicht nicht allen bekannt: Der METS Standard kennt sogenannte "Profiles", die auch bei der Library-of-Congress gehostet werden siehe [https://www.loc.gov/standards/mets/mets-registered-profiles.html](https://www.loc.gov/standards/mets/mets-registered-profiles.html).
Vielleicht ist da ja bereits etwas dabei, was in diesem Kontext hilfreich ist.
Und meiner Meinung nach sollte  versucht werden, das Ergebnis dieser Diskussion hier dort als  "Profile" zu registrieren.

**Reply by maria-federbusch** [reply]
Ich hoffe doch, dass wir das vorhandene Profil "METS-Anwendungsprofil für Digitalisierte Medien" entsprechend ergänzen und aktualisieren können. Es ist doch das Ziel, nur wirklich notwendige Regelungen für die Tonträger zu ergänzen - sie finden ja bereits Erwähnung auf verschiedenen Seiten. Nur leider fehlen komplexere Beispiele und Regelungen.

**Comment by maria-federbusch** [comment]
Hier ein Ausschnitt meiner METS-Dateien zur Diskussion, die aus meiner Sicht eine Format- und noch keine Kitododiskussion ist. Es handelt sich um eine Schellackplatte, auf beiden Seiten ist je ein Musikstück enthalten. Uns liegen Images der Label vor (als TIF, JPG) sowie Audiodateien (mp3, WAV, BWF). Ich habe auch USE-Attribute erfunden für die Audiomaster: AUDIOARCHIVE und AUDIOARCHIVE_SMALL.
```
<mets:structMap TYPE="LOGICAL" ><!--1 Schellackplatte, von der beide Seiten Ton enthalten, Typ B-->
        <mets:div CONTENTIDS="https://resolver.staatsbibliothek-berlin.de/SBB000356E300000000" ID="LOG_0000" DMDID="DMDLOG_0000" ADMID="AMD" LABEL="Berliner Luft" TYPE="sound_carrier" >
            <mets:div DMDID="DMDLOG_0001" ID="LOG_0001" LABEL="[A]" TYPE="section">
		<mets:div DMDID="DMDLOG_0002" ID="LOG_0002" TYPE="section" LABEL="Berliner Luft" />
            </mets:div>
	    <mets:div DMDID="DMDLOG_0003" ID="LOG_0003" LABEL="[B]" TYPE="section">
		<mets:div DMDID="DMDLOG_0004" ID="LOG_0004" TYPE="section" LABEL="Der Rixdorfer" />
            </mets:div>
		</mets:div>
    </mets:structMap>

    <mets:structMap TYPE="PHYSICAL">
	<mets:div ID="PHYS_0001" ORDER="1" TYPE="physSequence">
            <mets:div ID="PHYS_0002" ORDER="1" TYPE="track">
                <mets:fptr FILEID="FILE_0001_AUDIO"/>
                <mets:fptr FILEID="FILE_0001_AUDIOARCHIVE_SMALL"/>
		<mets:fptr FILEID="FILE_0001_AUDIOARCHIVE"/>
            </mets:div>
	   <mets:div ID="PHYS_0003" ORDER="2" TYPE="track">
                <mets:fptr FILEID="FILE_0002_AUDIO"/>
                <mets:fptr FILEID="FILE_0002_AUDIOARCHIVE_SMALL"/>
		<mets:fptr FILEID="FILE_0002_AUDIOARCHIVE"/>
           </mets:div>
	</mets:div>  
	<mets:div ID="PHYS_0004" ORDER="2" TYPE="physSequence">
              <mets:div ID="PHYS_0005" ORDER="1" TYPE="page">
                <mets:fptr FILEID="FILE_0001_DEFAULT"/>
                <mets:fptr FILEID="FILE_0001_THUMBS"/>
		<mets:fptr FILEID="FILE_0001_PRESENTATION"/>
            </mets:div>
            <mets:div ID="PHYS_0006" ORDER="2" TYPE="page">
                <mets:fptr FILEID="FILE_0002_DEFAULT"/>
                <mets:fptr FILEID="FILE_0002_THUMBS"/>
		<mets:fptr FILEID="FILE_0002_PRESENTATION"/>
            </mets:div>  
        </mets:div>
    </mets:structMap>
    <mets:structLink>
        <mets:smLink xlink:from="LOG_0000" xlink:to="PHYS_0001"/>
        <mets:smLink xlink:from="LOG_0000" xlink:to="PHYS_0002"/>
        <mets:smLink xlink:from="LOG_0000" xlink:to="PHYS_0003"/>
        <mets:smLink xlink:from="LOG_0000" xlink:to="PHYS_0004"/>
        <mets:smLink xlink:from="LOG_0000" xlink:to="PHYS_0005"/>
	<mets:smLink xlink:from="LOG_0000" xlink:to="PHYS_0006"/>
        <mets:smLink xlink:from="LOG_0001" xlink:to="PHYS_0005"/> <!--Seite A zu Labelimage 1-->
	<mets:smLink xlink:from="LOG_0002" xlink:to="PHYS_0002"/> <!--Stück 1 zu Audio 1-->
        <mets:smLink xlink:from="LOG_0003" xlink:to="PHYS_0006"/> <!--Seite B zu Labelimage 2-->
	<mets:smLink xlink:from="LOG_0004" xlink:to="PHYS_0003"/> <!--Stück 2 zu Audio 2-->	
    </mets:structLink>
```
